ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -60,9 +62,22 @@ import           Ouroboros.Network.Subscription.Dns
 import           Ouroboros.Network.Subscription.Worker (LocalAddresses (..))
 import           Ouroboros.Network.Tracers
 
+data DiffusionInitializationTracer
+  = RunServer
+  | RunLocalServer
+  | CreatingSystemdSocketForUnixPath !FilePath
+  | CreateSystemdSocketForSnocketPath !FilePath
+  | CreatedSystemdSocketForSnocketPath !FilePath
+  | BindingToSocket !FilePath !String
+  | ListeningToSocket !FilePath !String
```

Comment:
What about `ListeningLocalSocket`: by calling `listen` the socket is in listening mode or listening socket is a common phrase too.